src/backend/distributed/master/master_repair_shards.c

```diff
@@ -245,17 +245,22 @@ CopyDataFromFinalizedPlacement(Oid distributedTableId, int64 shardId,
 							   ShardPlacement *placementToRepair)
 {
 	char *relationName = get_rel_name(distributedTableId);
```

Comment:
Visual. Could you take this block under the next variable declaration block? We mostly do assignments here, and we use them later.